src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/StaffCategoryCaseTest.java

```diff
@@ -494,6 +496,167 @@ void shouldDeleteApprovedRequestedRoleForCase_V1_1() {
         });
     }
 
+    @Test
+    void shouldApproveCaseRequestedRoles_CCD_1_0_NotFromCCD() {
```

Comment:
Junit test name is misleading. It should be like shouldRejectCaseRequestedRolesForUnauthoriseRequest.